Makefile

```diff
@@ -31,7 +31,7 @@ test: clean
 	pip install -e . -r requirements/pytorch/devel.txt
 	pip install -r requirements/pytorch/strategies.txt
```

Comment:
shouldn't we also have `PACKAGE_NAME=pytorch` in front of those installs? If we don't, then `requirements/base.txt` will get changed and force `lightning_app` requirements, although they are not necessary. At least that's what happens for me (Python 3.9.13, pip 22.2.2)